justfile

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018-2020 Cargill Incorporated
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+splinter_repo := "../splinter"
+
+copy-docs:
+    #!/usr/bin/env sh
+    set -e
+    if test -f {{splinter_repo}}/Cargo.toml; then
```

Comment:
At this point, should just change this to the VERSION file.

Reply:
updated